src/app.js

```diff
@@ -0,0 +1,71 @@
+require("babel-runtime/regenerator");
+require("webpack-hot-middleware/client?reload=true");
+require('./favicon.ico');
+
+require("./scss/cart.scss");
+require("./scss/products.scss");
+require("./scss/login.scss");
+require("./scss/carousel.scss");
+require("./scss/query.scss");
+require("./scss/home.scss");
+require("./scss/common.scss");
+
+
+const header = `<div class="row">
+<nav>
+  <a href="#" class="logo">
+    <img src="./static/images/logo.png" alt="Sabka Bazaar" srcset="" class="logo">
+  </a>
+  <div>
+    <input type="checkbox" id="hamberger" />
+
+    <label class="nav_btn" for="hamberger">
+
+      <div class="center">
+        <div class="icon">
+          <span class="bar1"></span>
```

Comment:
Use a single span/i tag for defining the lines and use before and after pseudo elements instead of defining 3 different spans.

Reply:
Done